### PR TITLE
Feature/hw 67897 v3 java sdk user special status transition

### PR DESCRIPTION
**Pull request by jkurra-hw**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -236,7 +236,10 @@
                         <packageLineRate>85</packageLineRate>
                         <packageBranchRate>85</packageBranchRate>
                     </check>
-                    <format>xml</format>
+                    <formats>
+                        <format>xml</format>
+                        <format>html</format>
+                    </formats>
                     <maxmem>256m</maxmem>
```
Comment (arao6) [inline pom.xml:243]: Jayaram, could you please remove this change from pom.xml.

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java [hunk 2]
```diff
@@ -117,6 +98,46 @@ public void testGetUserStatusTransition() throws Exception {
...
+        assertThat(returnValue.getBusinessOperatingName(), is(equalTo("Big Baller Payments Operating")));
+        assertThat(returnValue.getDateOfBirth(), is(equalTo(dateFormat.parse("1980-01-01T00:00:00 UTC"))));
+        assertThat(returnValue.getGender(), is(equalTo(Gender.MALE)));
+        assertThat(returnValue.getPhoneNumber(), is(equalTo("647-90531")));
+        assertThat(returnValue.getMobileNumber(), is(equalTo("605-555-1323")));
+        assertThat(returnValue.getEmail(), is(equalTo("sf1601571120@sink.sendgrid.net")));
+        assertThat(returnValue.getGovernmentId(), is(equalTo("333333333")));
+        assertThat(returnValue.getEmployerId(), is(equalTo("222222222")));
+        assertThat(returnValue.getProgramToken(), is(equalTo("prg-eedaf875-01f1-4524-8b94-d4936255af78")));
+
```
Comment (arao6) [inline src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java:138]: pls remove this blank line

File: pom.xml (hunk 1 above)
Comment (jkurra-hw) [reply pom.xml]: Removed . Thanks @arao6

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java (hunk 2 above)
Comment (jkurra-hw) [reply src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java]: Done.

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java [hunk 3]
```diff
@@ -117,6 +98,45 @@ public void testGetUserStatusTransition() throws Exception {
         assertThat(returnValue.getToStatus(), is(equalTo(DE_ACTIVATED)));
     }
 
+    @Test
+    public void testUpdateUserVerificationStatus() throws Exception {
```
Comment (Rrathinasabapath) [inline src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java:102]: Please add IT cases for exceptions as mentioned in the page https://confluence.site1.hyperwallet.local/display/PF/Verification+enforcement+through+REST

**Review by Rrathinasabapath — CHANGES_REQUESTED** [review]
Please add exception test cases for IT.

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java (hunk 3 above)
Comment (jkurra-hw) [reply src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java:102]: added test cases . Thanks @Rrathinasabapath

**Review by arao6 — APPROVED** [review]

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java [hunk 4]
```diff
@@ -24,7 +24,7 @@
 
     public static enum Status {PRE_ACTIVATED, ACTIVATED, LOCKED, FROZEN, DE_ACTIVATED}
 
-    public static enum VerificationStatus {UNDER_REVIEW, VERIFIED, REQUIRED, NOT_REQUIRED}
+    public static enum VerificationStatus {UNDER_REVIEW, VERIFIED, REQUIRED, NOT_REQUIRED, REQUESTED}
```
Comment (Rrathinasabapath) [inline src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java:27]: Please add the missing statuses EXPIRED, READY_FOR_REVIEW and FAILED, run the tests again.

**Review by Rrathinasabapath — CHANGES_REQUESTED** [review]
Pls update the 'verificationStatus' value list.

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java (hunk 4 above)
Comment (jkurra-hw) [reply src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java]: added missing statuses and run the test cases working fine . Thanks @Rrathinasabapath

**Review by vkorotchenko — APPROVED** [review]

**Review by Rrathinasabapath — APPROVED** [review]
Changes look fine.

**Pull request merged**